moodle-tools/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Europe/Berlin
```

Comment:
i think this is not only in this Dockerfile, i think we once talked in a Reffinement about which system time we want to use, and did not we say that we want to use UTC?